lib/omise/OmiseCapabilities.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+class OmiseCapabilities extends OmiseApiResource
+{
+    const ENDPOINT = 'capability';
+
+    const FILTERS = [
```

Comment:
Since some of my other code already needs 5.4, which I assumed was ok was we are already recommending 5.6 on the readme and hence almost certainly dropping lower PHP versions soon